week-1/interpret/time-format.js

```diff
@@ -14,16 +14,38 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+// There are 7.
```

Comment:
This is not totally accurate.  the declaration `const` defines a constant which can only have one value assigned to it.  I would say these are constants and not true variables because their value cannt change.